recipes/bio-pyvol/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "bio-pyvol" %}
+{% set version = "1.7.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1d4b00c05f9f65d59e4d86170150e0bb8ad94057edcf4123488b1218862120a6
+
+build:
+  number: 0
+  entry_points:
+    - pyvol=pyvol.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - biopython >=1.73
+    - configparser
+    - msms >=2.6.1
+    - numpy >=1.16.1
+    - pandas >=0.24.1
+    - pip
+    - python
+    - scikit-learn >=0.20.2
+    - scipy >=1.2.1
+    - trimesh >=2.36.29
```

Comment:
Are most of these needed for building the package?

Reply:
I just did a more thorough re-read of the descriptions for the build, host, and run sections. I'm cleaning this up.